static/js/task-feed/controllers/task-feed.controller.js

```diff
@@ -27,8 +27,44 @@
       self.modules = [];
 
       TaskFeed.getProjects().then(
-        function success (data) {
-            self.modules = data[0];
+        function success (successData) {
```

Comment:
let's remove it when it's implemented and not wait for it. If you want it changed - let me know what logic is wrong it.